Makefile

```diff
@@ -1,2 +1,5 @@
 all:
-	docker-compose build --build-arg USER_ID=$(shell id -u) --build-arg GROUP_ID=$(shell id -g)
+	docker-compose build --build-arg USER_ID=$(shell id -u)
```

Comment:
Is it intentional that `--build-arg GROUP_ID=$(shell id -g)` is now gone?

Reply:
Yes, the group sometimes refuses to match on my side and I found that it works perfectly without it

Reply:
Alright, I was a bit confused since it's not mentioned in the commit message/description.